README.md

```diff
@@ -333,48 +339,54 @@ public interface ExampleHandler {
   // ...
 }
 ```
-You can declare your own methods that will be called in `onInvoke()` method of basic ManualHandler.
+You can declare your own methods that will be called in `onInvoke()` method of
+basic ManualHandler.
 
-##### Named
+##### NotNull
 
-Gets the field of the received packet by specified name. Can be multi-level.
+Checks if the needed value is not null before calling the handler method
```

Comment:
Let write it as 'Annotates that the method should only be called if all of its arguments are not null (in case of method-wide annotation) or only specific parameters are not null (in case of argument-wide annotations)'